dist/stampit.js

```diff
@@ -710,9 +738,13 @@ isStamp = function isStamp(obj) {
  * @return {Function}             A composable stampit factory
  *                                (aka stamp).
  */
-convertConstructor = function convertConstructor(Constructor) {
-  return stampit().methods(Constructor.prototype).enclose(Constructor);
-};
+function convertConstructor(Constructor) {
```

Comment:
Note that I replaced `.methods()` with `.state()`.

Does `.state()` follow the prototype chain or only work for own properties?